platform/mbed_retarget.cpp

```diff
@@ -147,6 +147,9 @@ DirectSerial::DirectSerial(PinName tx, PinName rx, int baud) {
     if (stdio_uart_inited) return;
     serial_init(&stdio_uart, tx, rx);
     serial_baud(&stdio_uart, baud);
+#if STDIO_UART_FC
```

Comment:
Looking at the other config there, should this become

`MBED_CONF_PLATFORM_STDIO_FC or FLOW_CONTROL` ?